dudes9/settings.py

```diff
@@ -75,8 +78,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.mysql',
+        'HOST': 'localhost',
```

Comment:
Don't commit credentials to VCS, even if it is local; Also, you may have problem if you will want to have different credentials for local and prod; Please use local_settings instead.